conans/client/toolchain/cmake/android.py

```diff
@@ -1,5 +1,130 @@
+import textwrap
+
 from .base import CMakeToolchainBase
 
 
 class CMakeAndroidToolchain(CMakeToolchainBase):
-    pass
+    _template_project_include = ''  # TODO: This file is not useful to Android, there is no MSVC runtime MD/MT
+
+    # TODO: Factorize with the native one
+    _template_toolchain = textwrap.dedent("""
```

Comment:
This is one of my concerns, that this diverge too early, and it is difficult to understand common parts and specialization parts, and fixes in other platforms fail to apply to others. 

Yes, please factorize this so we can better see isolated the differences of the Android toolchain.

Reply:
That's another PR to `develop` and yes, it is totally necessary (WIP). From this PR we should take some learnings about the toolchain feature, the same for the rest of PRs related to CMake toolchain. In this one I'm _learning_.

You will love how it looks like after the refactor 😄